royalty/queen

```diff
@@ -0,0 +1,4 @@
+Queen - (7)
+- If the King dies, a living Queen will become the new King. Prince takes priority over Queen if both are alive.
+- Upon death, if the cause of death is anything other than tribute, the Chief Warrior of their team will gain an extra attack the following night.
```

Comment:
Actually I have a question, is this power a 1-time thing?

Reply:
No, it is every time the Queen dies to anything other than tribute.

Reply:
Alright